projects/vehicles/worlds/highway_net/sumo.trip.xml

```diff
@@ -0,0 +1,3606 @@
+<?xml version="1.0"?>
```

Comment:
The `sumo.rou.xml` is the file needed to spawn and determined the path followed by the SUMO vehicles. So it is necessary.

It can be generated in two ways: 

- Either by generating a random `sumo.trip.xml`
- Or with a sumo.flow.xml` file.

When I regenerated the `sumo.rou.xml` file I used the first method because it seemed easier according to the doc. But I just had a look at the flow file that seems very simple and I will give it a go if it reduces the size of the `sumo.rou.xml`